example/react-native/.eslintrc

```diff
@@ -0,0 +1,9 @@
+{
+  "extends": "rallycoding",
```

Comment:
Is this still a leftover from a different example?

Reply:
Btw I think moving on to this new sdk it might be a nice opportunity to settle on one of the new opinionated tools that avoid jellyfishing, like using semistandard+prettier.

Reply:
No @pb82 I just liked this one and use it everywhere now